src/theme-cosmos.css

```diff
@@ -3,28 +3,92 @@ body {
   --ds-color: black;
   --ds-color-alt: white;
   --ds-color-primary: #5064FB;
-  --ds-font-family: "Gotham A", "Gotham B", "Gotham", sans-serif;
+  --ds-font-family: "GothamSSm", "Gotham SSm A", "Gotham SSm B", "Gotham A", "Gotham B", "Gotham", sans-serif;
```

Comment:
Remove `GothamSSm`, `Gotham A`, `Gotham B`, and `Gotham`. It would be good to add more fallback fonts too: `-apple-system,system-ui,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,sans-serif;`.

Note that currently Gotham SSm isn't running in production for cosmos.network yet. Therefore, this won't show the correct font, and will fallback to the next font. When we launch the redesign, we can switch to SSm in our typography.com account. 👍

Reply:
https://github.com/tendermint/cosmos-ui/pull/139/commits/3ed897802a226afa1c34191a3137768cd5856604